[debug] [2018-03-22T17:50:34.903Z] ----------------------------------------------------------------------
[debug] [2018-03-22T17:50:34.906Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\edlr7\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase serve
[debug] [2018-03-22T17:50:34.906Z] CLI Version:   3.17.7
[debug] [2018-03-22T17:50:34.906Z] Platform:      win32
[debug] [2018-03-22T17:50:34.906Z] Node Version:  v8.9.4
[debug] [2018-03-22T17:50:34.906Z] Time:          Thu Mar 22 2018 12:50:34 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:34.906Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-03-22T17:50:34.911Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-03-22T17:50:34.911Z] > authorizing via signed-in user
[debug] [2018-03-22T17:50:34.911Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2018-03-22T17:50:34.911Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 { refresh_token: '1/jSdTRTLe8GfAfDbcPo2IUKQ2BTXImExBxRne0d4kf1g',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  grant_type: 'refresh_token',
  scope: 'email https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloudplatformprojects.readonly https://www.googleapis.com/auth/firebase openid' } 
 Thu Mar 22 2018 12:50:34 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:35.342Z] <<< HTTP RESPONSE 200
[debug] [2018-03-22T17:50:35.358Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/project-adventure-7fb38  
  
 Thu Mar 22 2018 12:50:35 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:36.128Z] <<< HTTP RESPONSE 200
[debug] [2018-03-22T17:50:36.130Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/project-adventure-7fb38/tokens  
  
 Thu Mar 22 2018 12:50:36 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:36.503Z] <<< HTTP RESPONSE 200
[debug] [2018-03-22T17:50:36.505Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/project-adventure-7fb38  
  
 Thu Mar 22 2018 12:50:36 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:36.927Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from 'C:\Users\edlr7\desktop\project-adventure'...
[info] 
[debug] [2018-03-22T17:50:36.958Z] >>> HTTP REQUEST GET https://mobilesdk-pa.googleapis.com/v1/projects/1091190600610/clients/_:getWebAppConfig  
  
 Thu Mar 22 2018 12:50:36 GMT-0500 (Central Daylight Time)
[debug] [2018-03-22T17:50:37.464Z] <<< HTTP RESPONSE 200
[debug] [2018-03-22T17:50:37.522Z] TypeError: Cannot read property 'public' of undefined
    at Server.<anonymous> (C:\Users\edlr7\AppData\Roaming\npm\node_modules\firebase-tools\lib\serve\hosting.js:34:18)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Server.emit (events.js:208:7)
    at emitListeningNT (net.js:1378:10)
    at _combinedTickCallback (internal/process/next_tick.js:135:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
[error] 
[error] Error: An unexpected error has occurred.
